code/__DEFINES/construction/material.dm

```diff
@@ -1,11 +1,11 @@
 //Defines for amount of material retrived from sheets & other items
-/// The amount of materials you get from a sheet of mineral like iron/diamond/glass etc. 100 Units.
-#define SHEET_MATERIAL_AMOUNT 100
-/// The amount of materials you get from half a sheet. Used in standard object quantities. 50 units.
+/// The amount of materials you get from a sheet of mineral like iron/diamond/glass etc. 2000 Units.
+#define SHEET_MATERIAL_AMOUNT 2000
```

Comment:
why is this changed?

Reply:
The only thing using these defines are things added in plane cube anyway.